bleepy.py

```diff
@@ -483,7 +483,7 @@ def concat(self):
 
         blockfilename = self.getSaveDirectory()+"blocked"+str(uuid.uuid4())+"."+fileExt
 
-        txtconcat = "ffmpeg -safe 0 -f concat -i {} -c copy {}"
+        txtconcat = "ffmpeg -safe 0 -f concat -segment_time_metadata 1 -i {} -vf select=concatdec_select -af aselect=concatdec_select,aresample=async=1 {}"
```

Comment:
I tried this to solve the issue #5 but in the end it still output a video that is longer than the original